dev/package.sh

```diff
@@ -27,7 +27,7 @@ cd "$GLUTEN_DIR"
 
 # build gluten with velox backend, prompt always respond y
 export PROMPT_ALWAYS_RESPOND=y
-./dev/buildbundle-veloxbe.sh --enable_vcpkg=OFF --build_tests=ON --build_benchmarks=ON --enable_s3=ON  --enable_hdfs=ON "$@"
+./dev/buildbundle-veloxbe.sh --enable_vcpkg=OFF --build_tests=ON --build_benchmarks=ON --enable_s3=ON --enable_hdfs=ON build_arrow "$@"
```

Comment:
Adding `build_arrow` will cause only `build_arrow` command is executed in builddeps-veloxbe.sh. We should remove it.

https://github.com/apache/incubator-gluten/blob/main/dev/builddeps-veloxbe.sh#L308